### PR TITLE
boards: arm: stm32h750b_dk: add support for display (fixed)

### PR DESCRIPTION
**Pull request by toonst**
Add support for display on stm32h750b_dk. samples/drivers/display is compiling and flashing correctly now.  Previous PR is here: https://github.com/zephyrproject-rtos/zephyr/pull/67245 but, I can't reopen it.

Contains patch by [pcornu](https://github.com/pcornu), also tested by [ajarmouni-st](https://github.com/ajarmouni-st)

### CODE REVIEW

File: boards/st/stm32h750b_dk/stm32h750b_dk.dts
Comment (erwango) [inline boards/st/stm32h750b_dk/stm32h750b_dk.dts]: Question on these commented lies:
Will they be potentially useful ? 
   If yes, under which conditions ? -> A comment would be nice
  if not, can we remove them ?

File: boards/st/stm32h750b_dk/Kconfig.stm32h750b_dk
Comment (erwango) [inline boards/st/stm32h750b_dk/Kconfig.stm32h750b_dk:14]: Already included in `boards/arm/stm32h750b_dk/Kconfig.defconfig` doesn't look required here.

**Review by erwango — CHANGES_REQUESTED** [review]
The second commit doesn't look required.
Can you squash both ?

**Comment by ajarmouni-st** [comment]
Hi @toonst can you rebase & incoporate changes directly from my branch https://github.com/ajarmouni-st/zephyr_dev/tree/dev_stm32h750b_dk_display ? Thanks!

File: boards/arm/stm32h750b_dk/Kconfig.defconfig
Comment (mathieuchopstm) [inline boards/arm/stm32h750b_dk/Kconfig.defconfig:1]: This looks like a stray file (unless this is for HWMv1 compatibility?).
AFAIK, Kconfig should come from `boards/st/stm32h750b_dk/Kconfig.stm32h750b_dk` instead (which also contains this), so this file should probably be removed.

**Review by mathieuchopstm — COMMENTED** [review]
LGTM besides this:

File: boards/arm/stm32h750b_dk/Kconfig.defconfig
Comment (erwango) [reply boards/arm/stm32h750b_dk/Kconfig.defconfig:1]: Good catch this file is not required indeed. Please remove.

**Review by erwango — CHANGES_REQUESTED** [review]

**Comment by erwango** [comment]
I've pushed @ajarmouni-st 's branch here: https://github.com/zephyrproject-rtos/zephyr/pull/73107

**Comment by aescolar** [comment]
Bulk changing the milestone from everything which did not make it to the v3.7.0 freeze deadline
and which is not labeled as a bug or documentation (or does not appear clearly as only containing
a bug fix or a documentation change).
If you disagree about this change please add the milestone again.

Please note that until rc2 only PRs containing bug fixes, docs and tests can be merged.
After that and until release only bug fixes and docs changes.

Exceptions require approval by the release team and a vote by the TSC.
https://docs.zephyrproject.org/latest/project/release_process.html#stabilization-phase

**Comment by aescolar** [comment]
@toonst rebase needed

**Comment by ajarmouni-st** [comment]
Closed in favor of https://github.com/zephyrproject-rtos/zephyr/pull/74294.

**Pull request closed without merging**
